pkg/options/patchtmpl/patch_test.go

```diff
@@ -634,3 +670,59 @@ spec:
 		})
 	}
 }
+
+func TestConvertToFloat(t *testing.T) {
+	testCases := []struct {
+		desc   string
+		in     interface{}
+		exp    float64
+		expErr error
+	}{
+		{
+			desc: "float success",
+			in:   12.2,
```

Comment:
nit: wrap all the `in` cases with type conversions for clarity

Reply:
Done